server/src/main/java/org/opensearch/index/fielddata/ScriptDocValues.java

```diff
@@ -515,7 +515,7 @@ public void setNextDocId(int docId) throws IOException {
         }
 
         /**
-         * Set the {@link #size()} and ensure that the {@link #values} array can
+         * Set the {@link #size()} and ensure that the internal values array can
```

Comment:
`{@link #values}` - javadoc complains (for a reason) that `private final values` is not accessible, makes sense.